process/setup_sp.py

```diff
@@ -373,6 +373,100 @@ def process_distant_nodes(
     return(distant_nodes)
 
 
+#Cumulative opportunities (binary)
+#1 if d <= access_dist
+#0 if d > access_dist
+def binary_access_score(df, new_name, old_name, access_dist):
+    """
+    Calculate accessibiity score using binary measure: 1 if access <= access_dist, 0 otherwise
+
+    Parameters
+    ----------
+    df: DataFrame
+        DataFrame with origin-destination distances
+    new_name: list
+        list of new field names for new access score
+    old_name: list
+         list of original field names for distances
+    access_dist: int
+        access distance threshold
+
+    Returns
+    -------
+    DataFrame
+    """
+    df[new_name] = (df[old_name] <= access_dist).astype("Int64").fillna(0)

```

Comment:
The type could be `int` or `np.int64` rather than having to parse that string to a type.

Reply:
Come to think of this, why store this with 64-bits if the integer can only take on values of 0 or 1?

Reply:
It could be to handle nulls perhaps? (not sure if there should be valid cases of nulls, but if nulls are possible perhaps int64 is required?); eg as per https://pandas.pydata.org/pandas-docs/stable/user_guide/integer_na.html

Reply:
We're already calling `fillna`, so there should be no nulls. We can swap the `astype` and `fillna` order of operations if needed.

Reply:
As per the below comments that 'old_names' isn't required, the function could be simplified as:
```
def binary_access_score(df, distances, access_dist):
    """
    Calculate accessibiity score using binary measure: 1 if access <= access_dist, 0 otherwise
    Parameters
    ----------
    df: DataFrame
        DataFrame with origin-destination distances
    distances: list
         list of distances field names
    access_dist: int
        access distance threshold
    Returns
    -------
    DataFrame
    """
    scores = (df[distances] <= access_dist).astype("Int64").fillna(0)
    return scores
```
Using the dataframe constructed below to demonstrate functionality of the soft- and cumulative Gaussian functions, this looks like:
```
df[[f'bin_{x}' for x in distances]] = binary_access_score(df, distances, access_dist)
df
     x1    x2   soft_x1   soft_x2    cum_x1    cum_x2  bin_x1  bin_x2
0   100  1000  0.982014  0.006693  1.000000  0.082085       1       0
1   200   900  0.952574  0.017986  1.000000  0.201897       1       0
2   300   800  0.880797  0.047426  1.000000  0.406570       1       0
3   400   700  0.731059  0.119203  1.000000  0.670320       1       0
4   500   600  0.500000  0.268941  1.000000  0.904837       1       0
5   600   500  0.268941  0.500000  0.904837  1.000000       0       1
6   700   400  0.119203  0.731059  0.670320  1.000000       0       1
7   800   300  0.047426  0.880797  0.406570  1.000000       0       1
8   900   200  0.017986  0.952574  0.201897  1.000000       0       1
9  1000   100  0.006693  0.982014  0.082085  1.000000       0       1
```